modules/virt-deleting-vmis-web.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis.adoc
+
+[id="virt-deleting-vmis-web_{context}"]
+= Deleting a standalone virtual machine instance using the web console
+
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
```

Comment:
Request clarification. You want Workloads replaced by VirtProductName and Virtualization replaced by product-title? Did I get that right? By inserting these attributes/variables, the end result will show: "Workloads -> Virtualization" - is that correct?

Reply:
Nope, I just want `{VirtProductName} console` replaced with `{product-title} web console`, because the web console isn't the OpenShift virtualization console, it's the OpenShift Container Platform web console. The {product-title} variable expands to OpenShift Container Platform.

Reply:
Updated.